cmake/modules/LXQtTranslateTs.cmake

```diff
@@ -168,14 +168,16 @@ function(lxqt_translate_ts qmFiles)
         endif ()
         if (NOT EXISTS "${TR_TRANSLATION_DIR}/${TR_REPO_SUBDIR}")
             message(STATUS "Setting git repository in the translations dir '${TR_TRANSLATION_DIR}' ...")
+            if (EXISTS "${TR_TRANSLATION_DIR}/.git")
```

Comment:
This is needles, because with -f parameter the rm will not fail if the directory doesn't exist ... OK not needles, but not a "real optimization".